lib/builtin-unimport.t

```diff
@@ -0,0 +1,75 @@
+#!./perl
+
+BEGIN {
+    chdir 't' if -d 't';
+    require './test.pl';
+    set_up_inc('../lib');
+}
+
+# This file mostly contains tests of the PADNAMEf_TOMBSTONE mechanism.
+# Currently the only way it is accessible from perl code is via unimport from
+# builtin.
+
+use v5.36;
+no warnings 'experimental::builtin';
+
+# imported builtins can be unexported
+{
+    package UnimportTest;
+
+    sub true() { return "yes" };
+
+    {
+        use builtin 'true';
```

Comment:
you mentioned examples where `use builtin` should produce warning - this is one example

Reply:
Warning implemented - see #21915